Gruntfile.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+module.exports = function (grunt) {
```

Comment:
The build process is a left over from my changes. We may want to remove the build step altogether since it doesn't matter anymore. Whatever includes mux.js should have to worry about building it.